homeassistant/components/enocean/binary_sensor.py

```diff
@@ -3,12 +3,12 @@
 
 import voluptuous as vol
 
-from homeassistant.components import enocean
 from homeassistant.components.binary_sensor import (
     DEVICE_CLASSES_SCHEMA,
     PLATFORM_SCHEMA,
     BinarySensorEntity,
 )
+from homeassistant.components.enocean.device import EnOceanEntity
```

Comment:
Intra package imports should be relative.

Reply:
ouch, again. Corrected, and IDE configuration updated to avoid reproducing this error.